### ISSUE TITLE
Stale maven-wrapper.jar bundled with maven-wrapper-{version}.tar.gz since 0.1.4

### ISSUE THREAD

**Issue by mches** [takari/maven-wrapper#37]
When building `master`, which is currently at `0.1.7-SNAPSHOT`, a `maven-wrapper.jar` is magically installed in `target/maven-wrapper-0.1.7-SNAPSHOT/.mvn/wrapper/maven-wrapper.jar`. DEBUG logging doesn't indicate anything useful. The contents are always stale with a timestamp of `4/3/2015 1:07 AM`. 

META-INF/MANIFEST.MF:
```
Manifest-Version: 1.0
Implementation-Title: maven-wrapper
Implementation-Version: 0.1.2-SNAPSHOT
Implementation-Vendor-Id: io.takari
Build-Jdk: 1.7.0_67
Built-By: jvanzyl
Specification-Title: maven-wrapper
Created-By: Takari Inc.
Specification-Version: 0.1.2-SNAPSHOT
Archiver-Version: Provisio Archiver
```

This jar is added to `target/maven-wrapper-0.1.7-SNAPSHOT.tar.gz` which is later installed in the local repository. I've found that `target/maven-wrapper-0.1.7-SNAPSHOT.jar` and `target/maven-wrapper-0.1.7-SNAPSHOT-sources.jar` are not stale. The other 3 files in `target/maven-wrapper-0.1.7-SNAPSHOT.tar.gz` (`mvnw`, `mvnw.cmd` and `maven-wrapper.properties`) are not stale.

Incidentally, all versions since 0.1.2 (i.e. 0.1.4, 0.1.5 and 0.1.6) in Maven Central contain this stale jar.

**Comment by mches** [comment]
After doing all that analysis, it became pretty clear the bug was introduced [here](https://github.com/takari/maven-wrapper/commit/124c2cee6418be9740705ab726aabd264b0b45d6). The stale `maven-wrapper.jar` at `${basedir}/.mvn/wrapper/maven-wrapper.jar` was overwriting the project artifact for the purpose of assembling the `.tar.gz` distribution. A straight-forward fix is part of PR [#36](https://github.com/takari/maven-wrapper/pull/36).

**Comment by mosabua** [comment]
Committed fix, will release 0.2.0 shortly with fix

**Comment by mosabua** [comment]
Fyi maven-wrapper:0.2.0 and and takari-maven-plugin:0.4.0 are released to Central.
